### PR TITLE
Some improvements

### PR DESCRIPTION
**Pull request by julianduque**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test": "ava",
-    "watch": "ava --watch"
+    "test": "npm run lint && ava",
```
Comment (julianduque) [inline package.json:8]: @buritica this will add jscs to ci

File: package.json [hunk 2]
```diff
@@ -5,8 +5,9 @@
...
+    "watch": "ava --watch",
+    "lint": "jscs index.js lib test"
```
Comment (buritica) [inline package.json:10]: should we add `jscs` and `jshint` as devDependencies and then run them as`./node_modules/bin/jshint`?

File: package.json [hunk 3]
```diff
@@ -32,6 +33,7 @@
   "devDependencies": {
     "ava": "^0.14.0",
     "babel-preset-es2015": "^6.6.0",
+    "jscs": "^3.0.3",
```
Comment (buritica) [inline package.json:36]: oh, I see you added it here. will this `jscs` run as a 1st level command?

File: index.js [hunk 4]
```diff
@@ -52,15 +52,15 @@ controller.hears('invite', 'direct_mention', (bot, message) => {
 /**
  * Private Onboarding
  */
-controller.on('team_join', onboard);
+controller.on('team_join', () => setTimeout(onboard, 5000));
```
Comment (buritica) [inline index.js]: not sure this works this way, how are `bot`, `message` vars being passed to onboarding?

Perhaps we should add the delay in the actual onboarding script

File: package.json (hunk 3 above)
Comment (julianduque) [inline package.json:36]: when ran with `npm run` it will work

File: index.js (hunk 4 above)
Comment (julianduque) [inline index.js]: you are damn right, missed that part, in the actual onboarding script we need to add some sort of timeout to the tests too

Comment (buritica) [inline index.js]: Yes, can you restore this line and we'll do that one on a separate PR pls so we can merge the other stuff?
